CompatBaseline.txt

```diff
@@ -1,2 +1,8 @@
 # This API is marked obsolete and has been removed from the reference assembly
-MembersMustExist : Member 'Microsoft.Build.Utilities.ToolTask.EnvironmentOverride.get()' does not exist in the implementation but it does exist in the contract.
\ No newline at end of file
+MembersMustExist : Member 'Microsoft.Build.Utilities.ToolTask.EnvironmentOverride.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
@dsplaisted I updated the CompatBaseline file. Now there are no difference reported.
Does this look good to you?